docs/geomxngs_fastq_to_dcc.rst

```diff
@@ -0,0 +1,88 @@
+geomxngs_fastq_to_dcc
+-----------------------------------------------------------------------------------------
+Processes FASTQ sequencing files into digital count conversion (DCC) files using NanoString's GeoMx NGS Pipeline
+
```

Comment:
Add a note here: 

Non Broad Institute users that wish to run geomxngs_fastq_to_dcc must create a custom docker image that contains geomxngs. 

And add an instruction on how to build the docker (See [bcl2fastq](https://cumulus.readthedocs.io/en/stable/bcl2fastq.html#bcl2fastq-docker) instructions.)

May consider update the bcl2fastq-docker page to cover any private dockers.

See [bcl2fastq](https://cumulus.readthedocs.io/en/stable/bcl2fastq.html#bcl2fastq-docker) instructions.